layouts/partials/main_prefix.html

```diff
@@ -11,6 +11,7 @@
   SPDX-License-Identifier: EPL-2.0
 -->
 
+{{ partial "mobile-sidebar.html" . }}
```

Comment:
I am thinking that it would make more sense to move this under the H1.

Reply:
Emmmm.... But if some pages are hiding the page title and add title in the markdown content,  the collapsible menu will show above the title. 🤔 This might cause sometimes inconsistency where the menu shows.

Reply:
An example would be https://deploy-preview-174--eclipsefdn-hugo-solstice-theme.netlify.app/ 

and https://deploy-preview-174--eclipsefdn-hugo-solstice-theme.netlify.app/third-parent/

Reply:
If they are hiding the title, then there is just no title on the page...

Reply:
I also think we need to add a margin-bottom to this compoenet. There is no spacing between text and collapse menu.

Reply:
as discussed, bring back collapse menu above h1 title